package.json.unit.test.js

```diff
@@ -2,16 +2,13 @@ const packageLockJson = require('./package-lock.json')
 const packageJson = require('./package.json')
 
 describe('because rollup 0.60 drops support for Internet Explorer 8', () => {
-  describe('gulp-better-rollup', () => {
-    it('should be pinned to 3.1.0 in package.json', () => {
-      expect(packageJson.dependencies['gulp-better-rollup']).toEqual('3.1.0')
+  describe('rollup', () => {
+    it('should be pinned to 0.56.5 in package.json', () => {
+      expect(packageJson.dependencies.rollup).toEqual('0.56.5')
     })
-  })
 
-  describe('rollup', () => {
     it('should be pinned to 0.56.5 in package-lock.json', () => {
```

Comment:
Similar comment here in that we're probably able to bump this to 0.59.4.

Reply:
Consider it bumped 🎉